File: kata-solutions/video-store-kata/video-store-java/src/main/java/Movie.java
package com.cleancode.martinfowler.videostore;

public class Movie {
	
	private String title;
	private int priceCode;
	
	public Movie (String title) {
		this.title 		= title;
	}
	
	
	public String getTitle () {
		return title;
	}

  public double determineAmount(int daysRented) {
    return 0;
  }
  
  public int determineFrequentRenterPoints(int daysRented) {
    return 1;
  }

}

File: tdd-katas/video-store-kata/video-store-python/movie.py
from enum import Enum

class MovieType(Enum):
  CHILDRENS = 2
  REGULAR = 0
  NEW_RELEASE = 1

class Movie:
  def __init__(self, title, priceCode):
    self._title = title
    self._priceCode = priceCode
	
  def get_price_code (self):
    return self._priceCode
	
  def set_price_code (self, code):
    self._priceCode = code;
	
  def get_title (self):
    return self._title
